SDWebImageAVIFCoder/Classes/Private/Conversion.h

```diff
@@ -0,0 +1,15 @@
+//
+//  Conversion.h
+//  SDWebImageAVIFCoder
+//
+//  Created by psi on 2020/03/15.
+//
+
+#pragma once
+#if __has_include(<libavif/avif.h>)
+#import <libavif/avif.h>
+#else
+#import "avif/avif.h"
+#endif
+
+CGImageRef CreateCGImageFromAVIF(avifImage * avif);
```

Comment:
Same, can we name this `SDCreateCGImageFromAVIF` ?

Reply:
```c
extern SDCreateCGImageFromAVIF(avifImage *avif) __attribute__((visibility("hidden")));
```

Reply:
I added, thanks.